x-pack/plugins/infra/common/http_api/inventory_meta_api.ts

```diff
@@ -21,6 +21,7 @@ export const InventoryMetaResponseRT = rt.type({
 export const InventoryMetaRequestRT = rt.type({
   sourceId: rt.string,
   nodeType: ItemTypeRT,
+  currentTime: rt.number,
```

Comment:
now that we're moving to more exploration via the timeline, should we call this something like "selectedTime" maybe, to avoid confusion with this being "the current time on the clock right now"?

Reply:
We could... Do you want that changed in this PR or can we address that later when we change the time picker to provide a range instead of a single point?

Reply:
I feel like ... we probably won't remember to do it and this will sort of languish here as low-level brain-processing tech debt, but I don't feel strongly enough to hold up the PR. Up to you!

Reply:
I just did a quick search via `git grep -e \\WcurrentTime\\W x-pack/plugins/infra` which returned 79 instances. It might be better to follow up this PR with a seperate one.

Reply:
OK, that's fine. I doubt we need to change all of them, but we can look.